database/TAG1/POPULATE_DATA.dat.sql

```diff
@@ -0,0 +1,141 @@
+
+
+/*******************************************************************************
+ *
+ *	TICKET_STATUS
+ *
+ *******************************************************************************/
+INSERT INTO TICKET_STATUS VALUES (DEFAULT, 'New', 'New unassigned ticket' );
+INSERT INTO TICKET_STATUS VALUES (DEFAULT, 'Open', 'Assigned ticket' );
+INSERT INTO TICKET_STATUS VALUES (DEFAULT, 'Drop Off', 'A drop off that not assigned' );
+INSERT INTO TICKET_STATUS VALUES (DEFAULT, 'Pick Up', 'Ready for customer pick up' );
+INSERT INTO TICKET_STATUS VALUES (DEFAULT, 'Closed', 'Completed Ticket' );
+INSERT INTO TICKET_STATUS VALUES (DEFAULT, 'Cancelled', 'Cancelled Ticket' );
+
+
+/*******************************************************************************
+ *
+ *	TICKET_ACTION
+ *
+ *******************************************************************************/
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Install Filter', 	'Filters', 'Install a new filter on a device', '', 'Check if payment is required');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Adjust Filter',  	'Filters', 'Install a new filter on a device', '', '');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'App Lock', 		 	'Protection', 'Install a new filter on a device', '', '');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Block Texting', 	'Protection', 'Install a new filter on a device', '', '');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Block Texting and Web', 'Protection', 'Install a new filter on a device', '', '');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Chinuch', 			'Education', 'Install a new filter on a device', '', '');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Fix', 				'Other Service', 'Install a new filter on a device', '', '');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Information', 		'Education', 'Install a new filter on a device', '', '');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Pick Up', 			'Filters', 'Install a new filter on a device', '', '');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Remove Internet', 	'Protection', 'Install a new filter on a device', '', '');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Restrict Internet', 'Protection', 'Install a new filter on a device', '', '');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Update', 			'Protection', 'Install a new filter on a device', '', '');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Remove Video', 		'Protection', 'Install a new filter on a device', '', '');
+INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Remove Wifi and Video', 'Protection', 'Install a new filter on a device', '', '');
+
+
+/*******************************************************************************
+ *
+ *	AGENT
+ *
+ *******************************************************************************/
+INSERT INTO AGENT VALUES (DEFAULT, 'David', 'Smith', 'DSMITH', 'Password', '123 Main. St.', 'Lakewood', 'NJ', '08701', 'USA',
```

Comment:
Data is for testing. I hope some of the action and status types are correct.

 

From: nafg [mailto:notifications@github.com] 
Sent: Wednesday, June 17, 2015 4:38 AM
To: smith4312/TAG-Ticket
Cc: smith4312
Subject: Re: [TAG-Ticket] Db1 (#20)

 

In database/TAG1/POPULATE_DATA.dat.sql <https://github.com/smith4312/TAG-Ticket/pull/20#discussion_r32605177> :

> +INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Fix',                        'Other Service', 'Install a new filter on a device', '', '');
> +INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Information',                'Education', 'Install a new filter on a device', '', '');
> +INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Pick Up',                    'Filters', 'Install a new filter on a device', '', '');
> +INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Remove Internet',     'Protection', 'Install a new filter on a device', '', '');
> +INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Restrict Internet', 'Protection', 'Install a new filter on a device', '', '');
> +INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Update',                     'Protection', 'Install a new filter on a device', '', '');
> +INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Remove Video',               'Protection', 'Install a new filter on a device', '', '');
> +INSERT INTO TICKET_ACTION VALUES (DEFAULT, 'Remove Wifi and Video', 'Protection', 'Install a new filter on a device', '', '');
> +
> +
> +/*******************************************************************************
> + *
> + *   AGENT
> + *
> + *******************************************************************************/
> +INSERT INTO AGENT VALUES (DEFAULT, 'David', 'Smith', 'DSMITH', 'Password', '123 Main. St.', 'Lakewood', 'NJ', '08701', 'USA',

Is all of this data for testing purposes only?

—
Reply to this email directly or view it on GitHub <https://github.com/smith4312/TAG-Ticket/pull/20/files#r32605177> .  <https://github.com/notifications/beacon/ALe6bK3KCswBUKzjWohdu5OXE-0gpsSXks5oUSlZgaJpZM4FEcWB.gif>